src/index.css

```diff
@@ -13,3 +13,156 @@ body {
     'Helvetica Neue',
     sans-serif;
 }
+
+.book {
+  justify-content: space-around;
+  align-items: center;
+  border: 1px solid lightgray;
+}
+
+.nav-item {
+  list-style: none;
+  margin-right: 10px;
+  margin-top: 15px;
+  font-size: 15px;
+  text-transform: uppercase;
+}
+
+.nav-items {
+  margin-left: 35px;
+}
+
+.logo {
+  font-size: 35px;
+  color: #0290ff;
+  margin-left: 10px;
+}
+
+.form-on {
+  width: 40%;
+  height: 40px;
+}
+
+.add-head {
+  font-size: 35px;
+  margin-left: 10px;
+}
+
+.button {
+  width: 200px;
+  height: 40px;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.book-button {
+  border-right: 1px solid #e8e8e8;
+}
+
+.btn-chapter {
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+  font-size: 0.813rem;
+  letter-spacing: 0.5px;
+  text-align: center;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  width: 11.5rem;
+
+  /* height: 2.063rem; */
+  margin-top: 1.5em;
+}
+
+.cirPro {
+  border-right: solid 1px #e8e8e8;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5em;
+}
+
+.chap-1 {
+  color: #121212;
+  opacity: 0.5;
+  font-size: 0.813rem;
+}
+
+.chap-2 {
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #121212;
+  margin-top: 0.5em;
+}
+
+.form-container {
+  border-top: solid 1px #e8e8e8;
+  padding: 1.5em 0 10em 0;
+  margin-top: 1.5em;
+}
+
+.form-container p {
+  color: #888;
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+}
+
+.input-text {
+  border: solid 1px #e8e8e8;
+  padding: 0 0 0.2rem 1.063rem;
+  border-radius: 4px;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+}
+
+.title-input {
+  width: 50%;
+}
+
+.author-input {
+  width: 20%;
+}
+
+.input-submit {
+  width: 15%;
+  height: 2.813rem;
+  font-weight: bold;
+}
+
+.progress {
+  padding-left: 3em;
+  height: 3rem;
+}
+
+.perc {
+  color: #121212;
+  font-size: 2rem;
+}
+
+.completed {
+  color: #121212;
+  opacity: 0.5;
+  font-size: 0.875rem;
+}
+
+.oval {
+  border-image-source: linear-gradient(to bottom, var(--medium-blue), #379cf6);
+  width: 3.25rem;
+  height: 3.25rem;
+  border: 5px solid #379cf6;
+  border-radius: 50%;
+  color: #fff;
+}
+
+.book-container {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em;
+  border: 1px solid #e8e8e8;
+  border-radius: 4px;
+}
```

Comment:
- Please note that your design does not match the original Zeplin design. Kindly make sure your design looks similar (font, color, margin, etc.) before submitting again. 😉
- Original Design
![git](https://user-images.githubusercontent.com/44624138/201508190-d979fcee-8223-4783-a9ee-c19120a2de15.png)
- Your Design
![Screenshot from 2022-11-13 06-58-06](https://user-images.githubusercontent.com/44624138/201508216-83ad8999-fa20-492f-ac19-586707e6efed.png)